chain-storage-sqlite-old/benches/storage.rs

```diff
@@ -0,0 +1,56 @@
+#[macro_use]
+extern crate criterion;
+extern crate tempfile;
```

Comment:
do you still need to include `extern crate` ? This project is not edition 2018?

Reply:
Thanks for noting this, it doesn't really make sense here.